dist/mapdc.js

```diff
@@ -26020,7 +26020,7 @@ function getScales(_ref2, layerName) {
       range: color.range.map(function (c) {
         return (0, _utilsVega.adjustOpacity)(c, color.opacity);
       }),
-      default: (0, _utilsVega.adjustOpacity)("#27aeef", color.opacity),
+      default: (0, _utilsVega.adjustOpacity)(color.range[color.range.length - 1], color.opacity), // in current implementation 'Other' is always added as last element in the array
```

Comment:
This is better than an hardcoded value, but it will break if we change the logic that makes the "Other" color be the last of the color range. Still I suggest we keep it like this as it's simpler than other alternatives.

For example, just for the record, a more extensible way would be to repurpose the 
```
_chart.defaultColorAccessor = function() {
  return _defaultAccessor
}
```
from the color mixin that is not used anywhere else in the code (and even have a `// what is this?` comment right above). But it would involve bringing the color mixin to this raster mixin, which may just make it more confusing. So let's just leave it like this and revisit if needed.